Deprecations and index page review (#7)
neo4j/docs-graphql: modules/ROOT/pages/deprecations.adoc

[[Deprecations]]
= Deprecations

The following products and applications are deprecated:

- xref:deprecations.adoc#_grandstack_starter_app[GRANDstack starter app]
- xref:deprecations.adoc#_neo4j_graphql_architect[Neo4j GraphQL Architect]

== GRANDstack starter app

> The main purpose of the GRANDstack starter app to demonstrate how the Neo4j Labs GraphQL library could be used in the context of a full-stack application using React and Apollo client.

Reviewer: ```suggestion
The main purpose of the GRANDstack starter app was to demonstrate how the Neo4j Labs GraphQL library could be used in the context of a full-stack application using React and Apollo client.
```

Over time, the GRANDstack starter app grew to support other frameworks such as Flutter and Angular, thus the need to revisit its scope.
The intention is to replace this project with a new starter application product, which will focus on the backend and the configuration of the GraphQL library, as well as help developers with their front-end.

In the meantime, the `create-grandstack-app` npm package has been marked as deprecated.
It can still be used to skeleton a GRANDstack app, but the user will be warned that the package is deprecated.

The associated GRANDstack starter GitHub repository has been archived, meaning that it is still available as a reference, but is now read-only.
The associated npm packages (such as `graphql-auth-directives` and the `grandstack-cli`) have also been deprecated and archived on GitHub. 

== Neo4j GraphQL Architect

Neo4j GraphQL Architect was a low-code graph app for Neo4j Desktop that enabled developers to build, query, and deploy GraphQL APIs backed by the Neo4j graph database.
It was deprecated and removed from npm, the https://install.graphapp.io/[Graph App gallery], and Neo4j Desktop.
The associated GitHub repositories have also been archived.

> This tool was repurposed and expanded with xref:getting-started/toolbox.adoc[Neo4j GraphQL Toolbox], an onboarding, embeddable low-code tool that can be integrated to Neo4j.

Reviewer: This makes it sound like it was a rewrite, so I would maybe choose different wording here. Also remove "embeddable" again.

```suggestion
This tool was reimagined in the xref:getting-started/toolbox.adoc[Neo4j GraphQL Toolbox], an onboarding, low-code tool that can be integrated to Neo4j.
```
